main/lib/webApi.js

```diff
@@ -1,46 +1,39 @@
 import request from 'superagent'
 import { Dispatcher } from 'fluxxed_up'
 
-export function fetch(url, data, actionType) {
-  buildRequest(request.get(url), null, actionType)
-}
-
-export function post(url, data, actionType) {
-  buildRequest(request.post(url), data, actionType)
+function getErrors(res) {
```

Comment:
Changed the definition order--yes, yes, hoisting occurs, but this just helps the code read in a more-logical way.